### ISSUE TITLE
0.6.3 broken - "It may be due to an error while unencrypting password field"

### ISSUE THREAD

**Issue by jeekajoo** [cozy-labs/konnectors#233]
Hi,
Looks like it's konnectors (0.6.3) is broken on my side with
Data System: 2.5.5
Proxy: 2.5.6
Home: 2.5.9
Controller: 2.5.4

Logs
```
[2016-04-26 12:51:12:525] info - americano | add plugin: cozydb
[2016-04-26 12:51:12:529] info - Cozy DB | bill - byDate request creation...
[2016-04-26 12:51:12:579] info - Cozy DB | succeeded
[2016-04-26 12:51:12:579] info - Cozy DB | konnector - all request creation...
[2016-04-26 12:51:12:616] info - Cozy DB | succeeded
[2016-04-26 12:51:12:617] info - Cozy DB | bankoperation - byDate request creation...
[2016-04-26 12:51:12:647] info - Cozy DB | succeeded
[2016-04-26 12:51:12:647] info - Cozy DB | folder - byFullPath request creation...
[2016-04-26 12:51:12:704] info - Cozy DB | succeeded
[2016-04-26 12:51:12:704] info - Cozy DB | file - byFullPath request creation...
[2016-04-26 12:51:12:741] info - Cozy DB | succeeded
[2016-04-26 12:51:12:741] info - Cozy DB | steps - byDate request creation...
[2016-04-26 12:51:12:774] info - Cozy DB | succeeded
[2016-04-26 12:51:12:775] info - Cozy DB | sleep - byDate request creation...
[2016-04-26 12:51:12:814] info - Cozy DB | succeeded
[2016-04-26 12:51:12:814] info - Cozy DB | commit - byDate request creation...
[2016-04-26 12:51:12:864] info - Cozy DB | succeeded
[2016-04-26 12:51:12:864] info - Cozy DB | event - all request creation...
[2016-04-26 12:51:12:924] info - Cozy DB | succeeded
[2016-04-26 12:51:12:924] info - Cozy DB | event - bycaldavuri request creation...
[2016-04-26 12:51:12:985] info - Cozy DB | succeeded
[2016-04-26 12:51:12:985] info - Cozy DB | tag - byName request creation...
[2016-04-26 12:51:13:25] info - Cozy DB | succeeded
[2016-04-26 12:51:13:26] info - Cozy DB | cozyinstance - all request creation...
[2016-04-26 12:51:13:70] info - Cozy DB | succeeded
[2016-04-26 12:51:13:71] info - Cozy DB | user - all request creation...
[2016-04-26 12:51:13:136] info - Cozy DB | succeeded
[2016-04-26 12:51:13:136] info - Cozy DB | bankoperation - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | bill - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | binary - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | commit - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | contact - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | cozyinstance - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | event - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | file - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | folder - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | konnector - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | sleep - define indexes...
[2016-04-26 12:51:13:137] info - Cozy DB | steps - define indexes...
[2016-04-26 12:51:13:138] info - Cozy DB | tag - define indexes...
[2016-04-26 12:51:13:138] info - americano | cozydb loaded.
[2016-04-26 12:51:13:143] info - americano | Routes loaded.
[2016-04-26 12:51:13:145] info - americano | Configuration for production loaded.
konnectors server is listening on port 9358...
info - realtime-adapter | socket.io initialized
[2016-04-26 12:51:13:205] error - Attempt to retrieve password for birthdays failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:205] error - <encrypted_password_here>
[2016-04-26 12:51:13:205] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:205] error - Attempt to retrieve password for ical_feed failed: SyntaxError: Unexpected token Q
[2016-04-26 12:51:13:205] error - <encrypted_password_here>
[2016-04-26 12:51:13:205] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:205] error - Attempt to retrieve password for online_net failed: SyntaxError: Unexpected token Q
[2016-04-26 12:51:13:205] error - <encrypted_password_here>
[2016-04-26 12:51:13:205] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:205] error - Attempt to retrieve password for ameli failed: SyntaxError: Unexpected token B
[2016-04-26 12:51:13:205] error - <encrypted_password_here>
[2016-04-26 12:51:13:205] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:205] error - Attempt to retrieve password for kimsufi_ca failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:205] error - <encrypted_password_here>
[2016-04-26 12:51:13:207] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:207] error - Attempt to retrieve password for kimsufi_eu failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:207] error - <encrypted_password_here>
[2016-04-26 12:51:13:207] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:207] error - Attempt to retrieve password for ovh_ca failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:207] error - <encrypted_password_here>
[2016-04-26 12:51:13:207] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:207] error - Attempt to retrieve password for ovh_eu failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:207] error - <encrypted_password_here>
[2016-04-26 12:51:13:207] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:207] error - Attempt to retrieve password for runabove_ca failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:207] error - <encrypted_password_here>
[2016-04-26 12:51:13:207] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:207] error - Attempt to retrieve password for soyoustart_ca failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:207] error - <encrypted_password_here>
[2016-04-26 12:51:13:207] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:208] error - Attempt to retrieve password for soyoustart_eu failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:208] error - <encrypted_password_here>
[2016-04-26 12:51:13:208] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:208] error - Attempt to retrieve password for sncf failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:208] error - <encrypted_password_here>
[2016-04-26 12:51:13:208] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:208] error - Attempt to retrieve password for bouyguesbox failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:208] error - <encrypted_password_here>
[2016-04-26 12:51:13:208] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:209] error - Attempt to retrieve password for bouyguestelecom failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:209] error - <encrypted_password_here>
[2016-04-26 12:51:13:209] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:209] error - Attempt to retrieve password for digitalocean failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:209] error - <encrypted_password_here>
[2016-04-26 12:51:13:209] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:209] error - Attempt to retrieve password for electrabel failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:209] error - <encrypted_password_here>
[2016-04-26 12:51:13:209] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:209] error - Attempt to retrieve password for free failed: SyntaxError: Unexpected token Q
[2016-04-26 12:51:13:209] error - <encrypted_password_here>
[2016-04-26 12:51:13:209] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:209] error - Attempt to retrieve password for freemobile failed: SyntaxError: Unexpected token Q
[2016-04-26 12:51:13:209] error - <encrypted_password_here>
[2016-04-26 12:51:13:209] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:211] error - Attempt to retrieve password for github failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:211] error - <encrypted_password_here>
[2016-04-26 12:51:13:211] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:211] error - Attempt to retrieve password for githubcommits failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:211] error - <encrypted_password_here>
[2016-04-26 12:51:13:211] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:211] error - Attempt to retrieve password for isen failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:211] error - <encrypted_password_here>
[2016-04-26 12:51:13:211] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:211] error - Attempt to retrieve password for jawbone failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:211] error - <encrypted_password_here>
[2016-04-26 12:51:13:211] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:211] error - Attempt to retrieve password for nest failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:211] error - <encrypted_password_here>
[2016-04-26 12:51:13:211] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:211] error - Attempt to retrieve password for numericable failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:211] error - <encrypted_password_here>
[2016-04-26 12:51:13:212] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:212] error - Attempt to retrieve password for orange failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:212] error - <encrypted_password_here>
[2016-04-26 12:51:13:212] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:212] error - Attempt to retrieve password for rescuetime failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:212] error - <encrypted_password_here>
[2016-04-26 12:51:13:212] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:212] error - Attempt to retrieve password for sosh failed: SyntaxError: Unexpected token s
[2016-04-26 12:51:13:212] error - <encrypted_password_here>
[2016-04-26 12:51:13:212] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:212] error - Attempt to retrieve password for twitter failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:212] error - <encrypted_password_here>
[2016-04-26 12:51:13:212] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:212] error - Attempt to retrieve password for virginmobile failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:212] error - <encrypted_password_here>
[2016-04-26 12:51:13:212] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:212] error - Attempt to retrieve password for withings failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:212] error - <encrypted_password_here>
[2016-04-26 12:51:13:213] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:213] error - Attempt to retrieve password for googlecontacts failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:213] error - <encrypted_password_here>
[2016-04-26 12:51:13:213] error - It may be due to an error while unencrypting password field.
[2016-04-26 12:51:13:213] error - Attempt to retrieve password for linkedin failed: SyntaxError: Unexpected token x
[2016-04-26 12:51:13:213] error - <encrypted_password_here>
[2016-04-26 12:51:13:213] error - It may be due to an error while unencrypting password field.

/usr/local/cozy/apps/konnectors/build/server/init/konnectors.js:82
    log.info("Reseting isImporting field for " + slug + "...");
                                                 ^
ReferenceError: slug is not defined
    at konnectorResetValue (/usr/local/cozy/apps/konnectors/build/server/init/konnectors.js:82:50)
    at /usr/local/cozy/apps/konnectors/build/server/init/konnectors.js:61:18
    at /usr/local/cozy/apps/konnectors/node_modules/async/lib/async.js:181:20
    at Object.iterate [as _onImmediate] (/usr/local/cozy/apps/konnectors/node_modules/async/lib/async.js:262:13)
    at processImmediate [as _immediateCallback] (timers.js:336:15)
```
Let me know if you need more info.
Thanks

**Comment by frankrousseau** [comment]
We just published a new version. Can you try again?

**Comment by jeekajoo** [comment]
It is now fixed. Thanks

**Comment by clochix** [comment]
Thanks for reporting @jeekajoo, it should have been fixed by 7e64e38 in version 0.6.4, could you give it a try?
